tests/fixtures/transaction/error-teapot.xml

```diff
@@ -0,0 +1,23 @@
+POST https://api.recurly.com/v2/transactions HTTP/1.1
+X-Api-Version: {api-version}
+Accept: application/xml
+Authorization: Basic YXBpa2V5Og==
+User-Agent: {user-agent}
+Content-Type: application/xml; charset=utf-8
+
+<?xml version="1.0" encoding="UTF-8"?>
+<transaction>
+  <account>
+    <account_code>transactionmock</account_code>
+  </account>
+  <currency>USD</currency>
+  <amount_in_cents type="integer">1000</amount_in_cents>
+</transaction>
+
+HTTP/1.1 418 I'm a teapot
+Content-Type: application/xml; charset=utf-8
+
+<?xml version="1.0" encoding="UTF-8"?>
+<errors>
```

Comment:
Is this what our API returns or is the body empty?

Reply:
The API never returns a status 418 as far as I know, but this test is ensuring that the python client could handle any HTTP status code, including obscure ones such as 418.

The body itself was copied and pasted from a different test fixture. I can change that to be an empty body if that makes more sense. The body is not relevant to the test.